scripts/k8s/load-images-to-kind.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+# Print usage options for this script.
+print_help() {
+  cat <<EOF
+Usage: $(basename "${0}") --tag <TAG>
+
+Options:
+  -h, --help            Display this text.
+  --tag <TAG>           Input the container image tag.
+  --trim-debug-suffix   Remove the '-debug' suffix from image names.
+
+Examples:
+  $(basename "${0}") --tag "ribbit"
+EOF
+}
+
+SCRIPT_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]:-"$0"}")")"
+ROOT_DIR="$SCRIPT_DIR/../.."
+# Imports
+source "$ROOT_DIR/scripts/utils/log.sh"
+
+set -e
+
+TAG=
+TRIM_DEBUG_SUFFIX=0
+
+while test $# -gt 0; do
+  arg="$1"
+  case "$arg" in
+  --tag)
+    test $# -lt 2 && log_fatal "missing value for the argument '$arg'"
+    TAG=$2
+    shift
+    ;;
+  --tag=*)
+    TAG=${arg#*=}
+    ;;
+  --trim-debug-suffix)
+    TRIM_DEBUG_SUFFIX=1
```

Comment:
maybe adding -debug to begin with was a mistake, maybe need to consider removing it in the future

Reply:
Hmm, have to consider what was the reason for the debug. Maybe something depends on it?